packages/esm-patient-labs-app/src/config-schema.ts

```diff
@@ -16,7 +16,7 @@ export const configSchema = {
     _default: [
       {
         conceptUuid: 'ae485e65-2e3f-4297-b35e-c818bbefe894',
-        defaultOpen: true,
+        defaultOpen: false,
```

Comment:
@denniskigen do u have any objections to this? if not let's get it in.

Reply:
Hi @pirupius , the change looks good to go